src/app/cube/details/action-panel/action-panel.component.ts

```diff
@@ -251,6 +253,11 @@ export class ActionPanelComponent implements OnInit, OnDestroy {
     return this.windowWidth <= 750;
   }
 
+  openAdminApp() {
```

Comment:
We should add a JSDoc comment here to describe what purpose the function has and what it does.